drivers/bridge/bridge.go

```diff
@@ -295,6 +295,11 @@ func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) err
 	// Even if a bridge exists try to setup IPv4.
 	bridgeSetup.queueStep(setupBridgeIPv4)
 
+	enableIPv6Forwarding := false
+	if d.config != nil && d.config.EnableIPForwarding && config.FixedCIDRv6 != nil {
+		enableIPv6Forwarding = true
```

Comment:
Minor: would move 'd.config.EnableIPForwarding' down make it cleaner, so that ipv4 and ipv6 forwarding enabling go hand in hand:

`enableIPv6Forwarding = d.config.EnableIPForwarding`